src/main/resources/schemas/master_schema.json

```diff
@@ -881,16 +892,16 @@
                                               "title": "Information about linked cases",
                                               "type": "array",
                                               "items": {
-                                                "id": {
-                                                "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing/case/caseLinked/id",
-                                                "title": "Case Linked Id",
-                                                "description": "If a case is linked to another case, Id of that case will appear here.",
-                                                "type": "string",
-                                                "default": "",
-                                                "examples": [
-                                                  "offg35484"
-                                                ],
-                                                "pattern": "^.*$"
+                                                "caseId": {
```

Comment:
The change to caseId would technically be a breaking change

Would it be possible to keep this as ID?

Reply:
I think we cannot use ID as attribute, ID is reserved keyword to use in schema
I tried to use it as attribute, but it caused validation failure

Reply:
Following discussions happy that this needs to be changed to caseId